EX10.R

```diff
@@ -0,0 +1,52 @@
+## Exercise 10
+
+# Generate a line graph depicting the cumulative score for each team as a 
+#   function of time in the game
+
+setwd("/Users/bethoceguera/Documents/PhD Year 1/First Semester/Introduction to Biocomputing/Exercises/Exercise10")
+data <- read.table("UWvMSU_1-22-13.txt", head=TRUE,sep = "\t")
+library(ggplot2)
+
+#Define the working variable
+uwScores <- data[data$team == "UW",]
+msuScores <- data[data$team == "MSU",]
+
+#Table acummulating um scores
+for(i in 2:nrow(uwScores)){
+  uwScores[i,3] <- uwScores[i,3]+uwScores[(i-1),3]
+}
+
+#Table acummulating msu scores  
+for(i in 2:nrow(msuScores)){
+  msuScores[i,3] <- msuScores[i,3]+msuScores[(i-1),3]
+}
+
+#Plot with two lines for um and msu
+ggplot() + 
+  geom_line(data=uwScores,aes(x=time,y=score), color = "darkred") + 
+  geom_line(data=msuScores,aes(x=time,y=score), color="steelblue", linetype="twodash")
+
```

Comment:
an alternative way:
basketball_game <- read.table('UWvMSU_1-22-13.txt',header = TRUE, sep = '\t')


cumulative_score <- data.frame(matrix(0,nrow = 51,ncol = 4))
colnames(cumulative_score) <- c('UW', 'MSU', 'Half', 'Proportion')

for (i in 1:nrow(basketball_game)){

  if (basketball_game[i,2] == 'UW'){
    cumulative_score[i+1,1] <- basketball_game[i,3] + cumulative_score[i,1]
    cumulative_score[i+1,2] <- cumulative_score[i,2]
  }
  else if (basketball_game[i,2] == 'MSU'){
    cumulative_score[i+1,2] <- basketball_game[i,3] + cumulative_score[i,2]
    cumulative_score[i+1,1] <- cumulative_score[i,1]
  }
  if (basketball_game[i,1] < 20){
    cumulative_score[i+1,3] <- 1
  }
  else {cumulative_score[i+1,3] <- 2}

  cumulative_score[1,3] <- 1
}

cumulative_score[c(2:51),4] <- basketball_game$time/20